src/components/entity-info.component.js

```diff
@@ -96,7 +97,7 @@ export const EntityInfo = ({ entity, orgs, locale, navigation }: Props) => {
               type: 'octicon',
             }}
             subtitle={entity.location}
-            onPress={() => Communications.web(getLocationLink(entity.location))}
+            onPress={() => getUsersByLocation(entity.location, navigation)}
```

Comment:
Let's write the implementation here directly. `getUsersByLocation` sounds like returning users but it is not.